DataFormats/PatCandidates/interface/Jet.h

```diff
@@ -177,6 +177,20 @@ namespace pat {
     void scaleEnergy(double fScale) override { scaleEnergy(fScale, "Unscaled"); }
     void scaleEnergy(double fScale, const std::string& level);
 
+    // Methods to work with a JER correction factor
+    //
+    // is the JER correction factor valid, i.e. has it been written using the saveJerFactor method
+    bool isJerFactorValid() const;
+
+    // load the currently saved JER correction factor and check whether the factor was propely set before loading
+    float loadJerFactor() const;
+
+    // save a JER correction factor
+    void saveJerFactor(float jerFactor_);
+
+    // reset a saved JER correction factor
+    void resetJerFactor();
```

Comment:
I'm not quite sure where this is needed later?

Reply:
At the moment this is not needed. I just added it for convenience. In the new commit, it is still included, however I can also remove it if desired.

Reply:
I would just remove it to avoid confusion.

Reply:
Hi again,
I now implemented it in a way to better keep track whether the jerFactor is already set or not. With this the user should be aware that he overwrites an already existing factor and can then decide whether this is correct or not. If so, the resetJerFactor method can be used to be able to overwrite. Hope this is better.